cli/dcoscli/marathon/main.py

```diff
@@ -904,7 +920,14 @@ def pod_list(self, json_):
         self._ensure_pods_support(marathon_client)
 
         pods = marathon_client.list_pod()
-        emitting.publish_table(emitter, pods, tables.pod_table, json_)
+        queued_apps = marathon_client.get_queued_apps()
+        if json_:
```

Comment:
Yes, I had publish_table before. But I need more than one datasource now.
I have the list of the pods and the debugging information. publish table is only able to receive one data structure.

Alternative: should I enhance the pods json information about the `overdue` information per pod?